docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
+
+services:
+  mongo_host:
+    image: "mongo"
+    container_name: mongo_host
+    # restart: always
+    volumes:  
+      - dbdata:/data/db   
+    # networks:
+    #   - app-network 
+    ports:
+      - "37017:27017"
+  app:
+    image: express
+    container_name: express-mongo-host
+    build: .
+    restart: always
```

Comment:
restart always. That restarts each container?

Reply:
if you have it, otherwise, it would exit/stop